CHANGES.rdoc

```diff
@@ -3,6 +3,18 @@
 This document describes the relevant changes between releases of the
 _hawkular-client_ project.
 
+=== V 2.3.0
+
+* Wrap connection errors in `HawkularConnectionException` (a subclass of `HawkularException`)
+* Make sure the error callback is called on all kinds of errors (if it exists)
+* Use the new Rest-API of Hawkular-Inventory.
+* For authentication errors don't return raw html, but some sensible value
+* Make it explicit that rest-client v2.0 is used to follow ManageIQ
```

Comment:
this is not merged yet (I left a comment on that PR...)
btw, looking forward we can try using [this gem](https://github.com/skywinder/Github-Changelog-Generator) to get the list of changes automatically. wdyt?